Mage.Tests/src/test/java/org/mage/test/cards/watchers/IanMalcolmChaoticianTests.java

```diff
@@ -0,0 +1,61 @@
+package org.mage.test.cards.watchers;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestCommander4Players;
+
+/**
+ *
+ * @author jimga150
+ */
+public class IanMalcolmChaoticianTests extends CardTestCommander4Players {
+
+    @Test
+    public void testManaCostsandWatcher() {
+
```

Comment:
Library are shuffled by default at start of executing tests (after adding cards to library).
You need to call `skipInitShuffling();` would prevent that. (and then any `addCard` to library would put it on top.)